convert2rhel/subscription.py

```diff
@@ -973,8 +973,10 @@ def check_needed_repos_availability(repo_ids_needed):
         if repo_id not in avail_repos:
             # TODO: List the packages that would be left untouched
             loggerinst.warning(
-                "%s repository is not available - some packages"
-                " may not be replaced and thus not supported." % repo_id
+                "Some repositories are not available: %s. Some packages may not be replaced with"
```

Comment:
This warning is printed in a for loop meaning that if there's more repos not available the user would see multiple messages saying:
```
Some repositories are not available: repoA ...
Do you want to continue? Yes
Some repositories are not available: repoB ...
Do you want to continue? Yes
```
The plural there is weird when you go one by one. I'd be ok with the plural if you move the warning message outside of the for loop and list all the unavailable repos in it, like:
```
Some repositories are not available: repoA, repoB ...
Do you want to continue?
```

Reply:
Yeah that does sound weird when listing the repos like this, I will move the message out of the loop so that it just lists the repos out together!

Reply:
We've decided to remove the `check_needed_repos_availability()` later on because:
1. Shortly after this check function we move on to enabling RHSM repos (either the default ones from the the config or the ones specified through --enablerepo). And if these repos are not possible to enable, we stop the conversion.
2. Also, the check_needed_repos_availability checks only the default repoids, not the ones the user specified through --enablerepo.

For that reason, you can make just a minimal update to this one message, for example to have there:
```python
"The %s repository is not available. Some packages may not be replaced with their corresponding RHEL packages when converting." % repo_id
```